src/main/kotlin/org/jetbrains/plugins/bsp/magicmetamodel/impl/workspacemodel/impl/updaters/transformers/ModuleDetailsToGoModuleTransformer.kt

```diff
@@ -28,7 +28,7 @@ internal class ModuleDetailsToGoModuleTransformer(
     ResourcesItemToGoResourceRootTransformer(projectBasePath)
 
   override fun transform(inputEntity: ModuleDetails): GoModule {
-    val goBuildInfo = extractGoBuildTarget(inputEntity.target) ?: error("Transform error, cannot extract GoBuildTarget")
+    val goBuildInfo = extractGoBuildTarget(inputEntity.target)
```

Comment:
To w sumie nie wiem jak powinno byc, raz mi się tu wywaliło z tym errorem i się na niego wkurzyłem i usunąłem xD